docs/source/development/data_storage.rst

```diff
@@ -0,0 +1,24 @@
+.. _core-development:data-storage:
+
+*************
+Data storage
+*************
+
+
+.. _core-development:orm:database:
+
+Database schema & migrations
+=============================
+
+
+.. _core-development:orm:repository:
+
+File repository
```

Comment:
What will be the difference between this and the top level page "File repository"?

Reply:
It will explain the internals of the file storage mechanism (for the moment it's just the sharding, eventually this may include implementation details of the "disk object store")

Reply:
Sure, I was just wondering why there are _two_ section that are named "File Repository". This one, and another one in the file `docs/source/development/data_storege.rst`

Reply:
Ah, thanks, I guess Francisco forgot to delete the top-level section.